.github/workflows/scripts/buildkitenvs/buildx/default

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
do we want to align on sh or bash?

Reply:
There's nothing bashy-in here, really the line is to make my editor load the right toolset.
Otherwise totally fine to throw either sh or bash in there.